packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -12,6 +12,7 @@ import { IComboBoxOptionStyles, IComboBoxStyles } from './ComboBox.types';
 
 import { IButtonStyles } from '../../Button';
 import { memoizeFunction } from '../../Utilities';
+import { FontSizes } from '@uifabric/styling';

```

Comment:
Do not use this const. Use `theme.fonts.small`.
Also, I see you used `smallPlus`... that variable after the fluent type ramp update has the same value as small and should be avoided to be used even though it's the same value it juts brings mental confusion for whoever remembered that `small` is 12px and seeing `smallPlus` he would think that it's a bigger font size.

I really need to have the lint rule put in place to ban this import. (@KevinTCoughlin , @ecraig12345  FYI)

Reply:
Sorry about that i'll get the change in.